debian/clearwater-snmp-alarm-agent.postinst

```diff
@@ -60,6 +60,12 @@ case "$1" in
     abort-upgrade|abort-remove|abort-deconfigure)
     ;;
 
+    triggered)
+        # Restart the alarm agent. Use restart rather than stop
+        # as the alarm agent is covered by upstart not monit
+        service clearwater-snmp-alarm-agent restart || /bin/true
```

Comment:
Should be `restart clearwater-snmp-alarm-agent || /bin/true` since it's upstart (the `service` support is a shim).